js-miniapp-sdk/README.md

```diff
@@ -81,6 +81,24 @@ miniApp.requestLocationPermission()
 	});
 ```
 
+### 4. Show Interstitial Ad
+
+This is a helper function to make request to js bridge to show Iterstitial ads on the native SDK, The response is captured by a specific response type that is called InterstitialAdResponse and can be found here `src/types/responseTypes/interstitial/index.ts`.
+
+This is an intersection type consisting of a base interface called AdResponse can be found in  `src/types/responseTypes/AdResponse.ts` and InterStitial interface
```

Comment:
I think this line can be removed as it's implementation details: the user doesn't need to know this.

Reply:
Changed!